dapp/src/services/circulating-supply.service.js

```diff
@@ -0,0 +1,8 @@
+export default class CirculatingSupplyService {
+  async fetchCirculatingSupply() {
+    const endpoint = `/api/circulating-ogv`
```

Comment:
Let's not complicate this by adding another redirect here. This can directly call `api.originprotocol.com` instead of redirecting requests to ousd.com domain